project/src/components/reviewForm/reviewForm.tsx

```diff
@@ -0,0 +1,61 @@
+import {ChangeEvent, useState} from 'react';
+
+function ReviewForm () {
+
+  const MAX_RATING = 10;
+  const DEFAULT_RATING = Array(MAX_RATING).fill(null).map(()=>false);
+
+  const [rating, setRating] = useState(DEFAULT_RATING);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [review, setReview] = useState('');
+
+  const RATING_INPUTS = Array(MAX_RATING).fill(null).map((value, index)=> {
+    const INPUT_VALUE = (MAX_RATING-index).toString();
```

Comment:
```suggestion
    const INPUT_VALUE = (MAX_RATING - index).toString();
    
```

Отступ между оператором и операндом